app/src/app/components/view-profile/view-profile.component.html

```diff
@@ -0,0 +1,186 @@
+<div class="profile_container">
+  <div class="profile_row">
+    <div class="profile_title">
+      <mat-icon class="profile_icon" (click)="navigateToPersonalFiles()">arrow_back</mat-icon>
+      <h3 class="profile">{{'APP.EDIT_PROFILE.MY_PROFILE' | translate}}</h3>
+    </div>
+    <mat-divider class="mat_divider"></mat-divider>
+  </div>
+  <div class="profile_general_row" [formGroup]="profileForm">
+    <div class="profile_general">
+      <div class="profile_general_div">
+        <mat-icon class="profile_general_icon" (click)="toggle_general_dropdown()">
+          {{ general_section_dropdown ?  'expand_more' : 'chevron_right'}}</mat-icon>
+        <h4 class="general_title">{{'APP.EDIT_PROFILE.GENERAL' | translate}}</h4>
+      </div>
+      <div class="general_edit_btn" *ngIf="general_section_buttons_toggle">
+        <button mat-raised-button (click)="toggle_general_buttons()"
```

Comment:
incorrect naming convention in all the click handlers, instead of `toggle_general_buttons()` should be `toggleGeneralButtons()` instead (and in all other cases in this PR)

Reply:
Done.